bundles/ch.elexis.core.ui.mediorder/src/ch/elexis/core/ui/mediorder/AbstractBillAndCloseMediorderHandler.java

```diff
@@ -72,12 +162,15 @@ protected IStatus billAndClose(IModelService coreModelService, IContextService c
 				}
 			}
 		}
-
-		if (stock.getStockEntries().isEmpty() && removeStockEntry) {
-			coreModelService.remove(stock);
-	}
-
 		return Status.OK_STATUS;
 	}
 
-}
+	private Optional<ICoverage> getCoverage(IPatient patient, BillingLaw law) {
```

Comment:
Mach bitte folgendes: In `ch.elexis.covid.cert.ui.handler.CovidHandlerUtil#getCoverageWithLaw` liegt eine Implementation vor die ähnliches bezweckt. Verschieb diese in `ICoverageService` und schreib in `ICoverageServiceTest` einen TestCase dazu! Dann verwende hier und dort die Implementation aus dem `ICoverageService`